pyproject.toml

```diff
@@ -6,32 +6,33 @@ build-backend = "setuptools.build_meta"
 name = "mo-gymnasium"
 description="A standard API for MORL and a diverse set of reference environments."
 readme = "README.md"
-requires-python = ">= 3.9"
+requires-python = ">= 3.10"
 authors = [{ name = "Farama Foundation", email = "contact@farama.org" }]
 license = { text = "MIT License" }
 keywords = ["Reinforcement Learning", "Multi-Objective", "RL", "AI", "Gymnasium"]
 classifiers = [
     "Development Status :: 4 - Beta",  # change to `5 - Production/Stable` when ready
     "License :: OSI Approved :: MIT License",
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
     'Intended Audience :: Science/Research',
     'Topic :: Scientific/Engineering :: Artificial Intelligence',
 ]
 dependencies = [
     "gymnasium >=1.1.0",
-    "numpy >=1.21.0,<2.0",
+    "numpy >=1.21",
     "pygame >=2.1.3",
+    "pytest>=7.1.3",
```

Comment:
why adding this here if we have an extra for testing?

Reply:
True